main.cpp

```diff
@@ -385,95 +315,162 @@ Step 3 is written below UDTs 5 - 9
 */
 
 /*
-Thing 5)
+===================
+Part 1d assignment: Step 3
+===================
+You've just defined 10 UDTs!
+4 of them are unrelated (UDTs 1-4).
+5 of them form the sub-objects that make up the 10th UDT. 
+
+MOVE THEM BELOW this block comment and put them in numerical order 
+(1 - 10) starting on line 473
+    DO NOT COPY. 
+    I do not want to see 2 copies of your Plain-English UDTs.  
+    I only want to see the 10 UDTs written BELOW this block comment, in numerical order (1 - 10).
+
+After you have MOVED your 10 UDTs, send me a DM with your pull request link.
+I will review the pseudo-code that you have written.
+*/
+
+/*
+Thing 1) Guitar
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) number of strings
+    2) body wood type
+    3) brand name
+    4) body shape
+    5) size
 3 things it can do:
-    1)
-    2)
-    3)
+    1) play a chord
+    2) play a solo
+    3) connect to amplifier
  */
 
 /*
-Thing 6)
+Thing 2) wind mill
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) number of blades
+    2) rotational speed
+    3) pole height
+    4) amount of energy generated in a day
+    5) blade length
 3 things it can do:
-    1)
-    2)
-    3)
+    1) convert mechanical energy into electrical energy
+    2) rotate their baldes
+    3) connect to the electrical grid
  */
 
 /*
-Thing 7)
+Thing 3) Motorcycle
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) RPMs
+    2) color
+    3) size
+    4) engine type
+    5) brand name
 3 things it can do:
-    1)
-    2)
-    3)
+    1) accelerate
+    2) consume gas
+    3) cruise
  */
 
 /*
-Thing 8)
+Thing 4) Drill set
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) drill bit material
+    2) drill bit diameter
+    3) angular speed
+    4) number of chargers
+    5) brand name
 3 things it can do:
-    1)
-    2)
-    3)
+    1) drill concrete
+    2) unscrew in the left or right direction
+    3) set rotatory speed
```

Comment:
You need to provide the type you'll use for each of these 5 properties
- [ ] revise